asreviewcontrib/insights/utils.py

```diff
@@ -15,3 +40,21 @@ def get_labels(state_obj, priors=False):
         labels = labels + np.zeros(n_records - len(labels)).tolist()
 
     return labels
+
+
+def iter_states(file_paths):
```

Comment:
Also private?

Reply:
I noticed when making the API example for multiple curves with a custom legend, that I wanted to use this function, but it was private. Maybe there's an argument for keeping this public, though I agree that you don't want people to use this function outside of this specific case. So I made it private in the update.